roles/dtc/common/templates/ndfc_vrf_lite.j2

```diff
@@ -1,6 +1,6 @@
 {% if item.ospf is defined and item.ospf %}
-{% include 'ndfc_vrf_lite_ospf.j2' %}
+  {% include 'ndfc_vrf_lite_ospf.j2' %}
```

Comment:
Can we change this line to:

```{% include 'ndfc_vrf_lite/ndfc_vrf_lite_ospf.j2' %}```

This means you need to create an ```ndfc_vrf_lite``` directory under ```templates``` which aligns with how we've started separating things in already committed and subsequent PRs.

Reply:
done, please have a look

Reply:
Thanks @GaetanoCarlucci! Can you resolve the conversations as you complete them as it is a check done in GitHub that needs to be done to enable merging the PR.